packages/mux-player/README.md

```diff
@@ -198,6 +198,18 @@ Instead, some components expose **parts** that can be targeted with the [CSS par
 <mux-player playback-id="DS00Spx1CV902MCtPj5WknGlR102V5HFkDe"></mux-player>
 ```
 
+### Controls Backdrop Color
+
+We expose a CSS variable to set the controls backdrop color.
+
+```css
+mux-player {
+  --controls-backdrop-color: rgb(0 0 0 / 0%);
+}
+```
+
+Turning this off completely has implications on the accessibility of the controls as they may not meet [the contrast ratio requirements for WCAG 2.1](https://www.w3.org/TR/WCAG/#contrast-minimum) without it.
```

Comment:
suggestion(non-blocking)
> Turning this off completely or adjusting it significantly can effect accessibility considerations and compliance for [the contrast ratio requirements ...

(or similar)
Also, in my meeting w/the a11y audit company, they let me know that https://www.w3.org/TR/WCAG22/ just happened (which they tested against, iirc)

Reply:
Looks like 2.2 is still in CR and not a TR just yet, but I do think it'll be officially official 🔜